SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py

```diff
@@ -233,11 +233,17 @@
         AddNoisyPixels = False,
         AddInefficiency = False,
         AddThresholdSmearing = False,
+        Phase2ReadoutMode = -1,
+        AdcFullScale = 255,
+        AddXTalk = False,
```

Comment:
I would recommend documenting this here:
https://github.com/cms-sw/cmssw/blob/f738feebb2f2a520baac4e21cd3da20e3e52236e/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py#L219-L223